### PR TITLE
broke my methods

### PR DESCRIPTION
**Pull request by apancorb**

### CODE REVIEW

File: Projects/P1/src/Map.java [hunk 1]
```diff
@@ -66,26 +66,26 @@ public boolean move(String name, Location loc, Type type) {
 			field.get(old).remove(type);
 			field.get(loc).add(type);
 			// if all goes fine return true
-			return true;
+			return false; // <-- error there
```
Comment (bhegde8) [inline Projects/P1/src/Map.java:69]: You should probably remove the <-- error there comment

**Review by bhegde8 — CHANGES_REQUESTED** [review]

File: Projects/P1/src/Map.java (hunk 1 above)
Comment (apancorb) [reply Projects/P1/src/Map.java:69]: lol aight just trying to be nice

**Comment by apancorb** [comment]
updated it

**Review by bhegde8 — CHANGES_REQUESTED** [review]
Just tested. It turns out that although TestMapMove now fails, TestGhostValidMoves and TestPacManValidMoves still pass despite this sabotage. Those tests will only fail if they get an empty ArrayList.

**Comment by apancorb** [comment]
aight i will just return an empty array

**Review by bhegde8 — APPROVED** [review]
Looks good to me

**Pull request merged**